.github/workflows/ci.yaml

```diff
@@ -216,26 +216,44 @@ jobs:
       JAX_IMAGE: ${{ needs.build-jax.outputs.DOCKER_TAGS }}
     secrets: inherit
 
-  test-te:
-    needs: build-te
-    uses: ./.github/workflows/_test_te.yaml
-    with:
-      JAX_TE_IMAGE: ${{ needs.build-te.outputs.DOCKER_TAGS }}
-    secrets: inherit
-  test-t5x:
+  # test-te:
```

Comment:
How about a comment here to inform why this was disabled?

Reply:
This was just a temporary WAR to the issue of having too many unique workflows. We want to comment this back in prior to merging